### PR TITLE
[ja] In-page links are not working etc in /ja/docs/tasks/debug/debug-application/debug-pods.md

### PR DESCRIPTION
**Pull request by K0more**
PR for issue #37351 
fixed the following things:
- In-page links
- bold notations
- the required ja to eng edits.

signed-off-by: k0more arch.phenon@gmail.com

### CODE REVIEW

**Comment by K0more** [comment]
can you add the hacktoberfest label to this PR as well, it'd be really appreciated.

**Review by t-inu — CHANGES_REQUESTED** [review]
This file is described using Markdown format.
[Hugo](https://gohugo.io/) generates a HTML file from this md file.
So, need not changing to the HTML format in this PR.

And, how to set custom heading IDs, refer to the following.
https://gohugo.io/content-management/cross-references/#heading-ids

File: content/ja/docs/tasks/debug/debug-application/debug-pods.md [hunk 1]
```diff
@@ -134,7 +133,7 @@ Podの`containerPort`がServiceの`targetPort`と一致することを確認し
 
 詳しくは[Serviceのデバッグ](/ja/docs/tasks/debug/debug-application/debug-service/)を参照してください。
 
-## {{% heading "whatsnext" %}}
+## 次の項目
```
[the remark this answers is not in the document]
Comment (K0more) [reply content/ja/docs/tasks/debug/debug-application/debug-pods.md:136]: okay 👍

**Comment by K0more** [comment]
@t-inu Hey, I have updated the md file, can you recheck it.

**Comment by nasa9084** [comment]
/assign
/approve

btw we don't have hactoberfest labels

**Comment by K0more** [comment]
> /assign /approve
> 
> btw we don't have hactoberfest labels

Oh ok no problem, I was just curious

**Comment by t-inu** [comment]
Thanks for the fix.
/label tide/merge-method-squash
/lgtm

**Comment by K0more** [comment]
たつけて ありがとうございます @t-inu @nasa9084

**Pull request merged**